doc/users/plotting/examples/colormap_normalizations_arbitrarynorm.py

```diff
@@ -0,0 +1,62 @@
+"""
+Demonstration of using the ArbitraryNorm classes for normalization.
+"""
+
+import numpy as np
+import matplotlib.colors as colors
+import matplotlib.pyplot as plt
+import matplotlib.cm as cm
+
+x=np.linspace(0,16*np.pi,1024)
```

Comment:
Ok thanks! I was not sure if the PEP8 compatibility also applied to the examples. I will correct this.